### ISSUE TITLE
Build NOTE: use of unexported objects

### ISSUE THREAD

**Issue by jacobvjk** [RMI-PACTA/pacta.aggregate.loanbook.plots#100]
```
Unexported objects imported by ':::' calls:
    ‘r2dii.plot:::abort_if_missing_names’
    ‘r2dii.plot:::abort_if_unknown_values’
    See the note in ?`:::` about the use of this operator.
```

@jdhoffa @MonikaFu maybe we can either export these functions in r2dii.plot or put them elsewhere to avoid duplication and sketchy dependencies?

**Comment by jdhoffa** [comment]
These functions are pretty simple (i don't think they warrant a whole new package).
I would just write them out again in this package and maintain them here

**Comment by MonikaFu** [comment]
I agree with Jackson. Probably the best solution as they are not the core functionality of `r2dii.plot` but helper functions.
